### PR TITLE
Chasms properly smooth again

### LINKED ISSUE #25140: Chasms not smoothing properly
I believe it was tendril created one.

dreamseeker_2017-03-15_18-43-34

### PR DESCRIPTION
**Pull request by Supermichael777**
:cl:
fix: Chasms now smooth properly.
/:cl:

Effectively every chasm that is actually mapped or created is a sub-type and smoothing recognizes them as different if you don't set the SMOOTH MORE flag. So because the types weren't in the list it didn't smooth.

Fixes #25140

### CODE REVIEW

**Comment by Cyberboss** [comment]
Fixes #25140

File: code/game/turfs/simulated/chasm.dm [hunk 1]
```diff
@@ -5,10 +5,10 @@
 	name = "chasm"
 	desc = "Watch your step."
 	baseturf = /turf/open/chasm
-	smooth = SMOOTH_TRUE | SMOOTH_BORDER
+	smooth = SMOOTH_TRUE | SMOOTH_BORDER | SMOOTH_MORE
 	icon = 'icons/turf/floors/Chasms.dmi'
 	icon_state = "smooth"
-	canSmoothWith = list(/turf/open/floor/carpet, /turf/open/chasm)
```
Comment (Cyberboss) [inline code/game/turfs/simulated/chasm.dm]: Why'd you remove carpet smoothing?

Comment (ChangelingRain) [inline code/game/turfs/simulated/chasm.dm]: fake pits aren't subtypes of carpets

Comment (Cyberboss) [inline code/game/turfs/simulated/chasm.dm]: But isn't it so we don't get carpet edges when a pit appears in the middle?

Comment (ChangelingRain) [inline code/game/turfs/simulated/chasm.dm]: why do carpets smooth with chasms fake pits are not subtypes of carpets

Comment (RandomMarine) [inline code/game/turfs/simulated/chasm.dm]: Because #24660

Comment (ChangelingRain) [inline code/game/turfs/simulated/chasm.dm]: fake pits _are not subtypes of carpets_

Comment (xxalpha) [inline code/game/turfs/simulated/chasm.dm]: This is all wrong, the problem isn't on the smoothing configuration of the chasms.

File: code/game/turfs/simulated/chasm.dm [hunk 2]
```diff
@@ -5,10 +5,10 @@
...
+	canSmoothWith = list(turf/open/floor/fakepit, /turf/open/chasm)
```
Comment (lzimann) [inline code/game/turfs/simulated/chasm.dm]: forgot a `/` in the start

File: code/game/turfs/simulated/chasm.dm (hunk 1 above)
Comment (Supermichael777) [inline code/game/turfs/simulated/chasm.dm]: if you look out the right window you will see that this line was altered but github handles that as add and remove

**Pull request merged**